### PR TITLE
#168781612 - Setting up React project

### PR DESCRIPTION
**Pull request by ngireric123**
#### What does this PR do?
Setting up React project for barefoot nomad frontend.

#### Description of Task to be completed?
- add .gitignore file
- Add .babelrc file
- Add islintignore file
- Add web pack.config file
- Add src folder
- install webpack 
- install babel
- eslint

#### How should this be manually tested?
- npm install
- npm start
- Then browser open itself
#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#168781612](https://www.pivotaltracker.com/story/show/168781612)

#### Screenshots (if appropriate)
image

Questions:
N/A

### CODE REVIEW

**Review by sabin18 — APPROVED** [review]
@ngireric123  good job

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App';
+
+ReactDOM.render(<App />, document.getElementById('app'));
```
Comment (nignanthomas) [inline src/index.js]: Kindly add a new line at the end of the file.

**Review by nignanthomas — CHANGES_REQUESTED** [review]
Great job @ngireric123 
I left a comment. Kindly implement it.
And edit the PR's title to match conventions.

**Review by nignanthomas — APPROVED** [review]

**Review by Cheza-Dzabala — APPROVED** [review]
Good job

**Review by alainmateso — APPROVED** [review]

**Pull request merged**